deployments/local/ops/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+services:
+  acme-grafana:
+    image: grafana/grafana:8.0.2
+    ports:
+      - 3000:3000
+    user: "104:1000"
+    volumes:
+      - ./run/grafana:/var/lib/grafana
+      - ../../../config/stage/ops/grafana/defaults.ini:/usr/share/grafana/conf/defaults.ini
+
+  acme-prometheus:
+    image: prom/prometheus:v2.28.0
+    user: "65534:1000"
+    volumes:
+      - ../../../config/stage/ops/prometheus/:/etc/prometheus/
+      - ./run/prometheus:/prometheus
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus'
+      - '--web.console.libraries=/usr/share/prometheus/console_libraries'
+      - '--web.console.templates=/usr/share/prometheus/consoles'
+
+  acme-loki:
+    image: grafana/loki:2.0.0
+    ports:
+      - "3100:3100"
+    command: -config.file=/etc/loki/local-config.yaml
+#    volumes:
+#      - ../../../config/stage/ops/loki/:/etc/loki/
+
+  acme-promtail:
+    image: grafana/promtail:2.0.0
```

Comment:
Is it possible to feed Keycloak logs just from the container stdout, e.g. by configuring a proper log driver for the container instead of writing a log file?

Reply:
Seems to be open on promtail: https://github.com/grafana/loki/issues/3374

And for sure this could be a reason to skip promtail and try something different....

Reply:
Ah there is one solution described using the docker logs